examples/lighting-app/ameba/chip_main.cmake

```diff
@@ -192,6 +192,7 @@ target_include_directories(
     ${chip_dir}/zzz_generated/lighting-app
     ${chip_dir}/zzz_generated/lighting-app/zap-generated
     ${chip_dir}/zzz_generated/app-common
+    ${chip_dir}/examples/all-clusters-app/ameba/main/include #to get AmebaObserver.h
```

Comment:
having `examples/lighting-app` point to `examples-all-clusters-app` does not look right. Please figure out another way to layer things that makes sense. Several platforms have common or maybe some ameba/observer would be appropriate

Reply:
Updated in the last commit, please re-review.